### DISCUSSION TITLE
pydantic-core

### DISCUSSION THREAD

**Discussion by FeodorFitsner** [flet-dev/flet#3141] [Packages] [answered]
Add `pydantic-core` package to Flet.

**Comment by 7576457** [comment]
yes! How many votes are needed to add this or that dependency?

**Reply by FeodorFitsner** [reply]
it's been implemented already. We are going to release a new packaging soon using it.

**Reply by 7576457** [reply]
#3425 Are websockets implemented?

**Reply by FeodorFitsner** [reply]
yep: https://pypi.flet.dev/simple/

new packaging (without Kivy) uses that feed.
Will release a new serious_python hopefully this week.

**Reply by reriley** [reply]
Has the newly implemented version been released? I'm trying to package an app for Windows but am confused by how non-pure Python packaging works. Kivy and p4a are talked about a lot but seem to be for Android only.

**Reply by FeodorFitsner** [reply]
Working on it...and making really good progress for both iOS and Android! There were some nuances with iOS, but it's done now.
We hope to roll out a pre-release with a new packaging in the end of next week.
For Windows: you don't need anything special to make Windows work - most of packages on pypi already offer Windows wheels, including [pydantic-core](https://pypi.org/project/pydantic-core/2.24.0/#files).

**Reply by reriley** [reply]
Thank you. I must have a different problem then. I found no answer so created a question here: https://github.com/flet-dev/flet/discussions/4035

BTW, thank you so much for this package and the incredible support and continuous improvement!

**Reply by OtherwisePitt92** [reply]
> Working on it...and making really good progress for both iOS and Android! There were some nuances with iOS, but it's done now. We hope to roll out a pre-release with a new packaging in the end of next week. For Windows: you don't need anything special to make Windows work - most of packages on pypi already offer Windows wheels, including [pydantic-core](https://pypi.org/project/pydantic-core/2.24.0/#files).

the problem is for web

**Comment by FeodorFitsner** [comment, marked as answer]
[`pydantic-core`](https://pypi.flet.dev/pydantic-core) and [other packages](https://pypi.flet.dev/) are available in Flet 0.25.0 pre-release and up for both iOS and Android. [Give it a try](https://flet.dev/blog/flet-new-packaging-pre-release) and let us know how that works.

**Comment by Jzhenli** [comment]
@FeodorFitsner would u offer pydantic-core for python3.10 or python3.11? Thanks.
